app/serializers/concept_serializer.rb

```diff
@@ -0,0 +1,24 @@
+class ConceptSerializer < ActiveModel::Serializer
+  attributes :id, :name, :uid, :parent_id, :level, :parent_uid
+  delegate :cache_key, :to => :object
+
+  def to_json(*args)
+    Rails.cache.fetch expand_cache_key(self.class.to_s.underscore, cache_key, 'to-json') do
```

Comment:
@kriskelly if we are able to pull in the parent_uid with SQL, then a cache at the individual level would no longer be necessary